src/aspire/source/xform.py

```diff
@@ -219,7 +219,7 @@ def _forward(self, im, indices):
             idx_k = np.where(self.filters[indices] == f)[0]
             flip_filter = LambdaFilter(f, np.sign)
             if len(idx_k) > 0:
-                im_out[:, :, idx_k] = Image(im_in[:, :, idx_k]).filter(flip_filter).asnumpy()
+                im_out[idx_k] = Image(im_in[idx_k]).filter(flip_filter).asnumpy()
```

Comment:
it can, but I feel like rewriting that sorf of code (its not just here...) is out of scope for this PR. This change appears to correctly perform the col to row conversion.  I find the filter and transform codes to be way over complicated for what they are supposed to do for my taste, and I think it would be a rabbit hole... can we punt on that?

Reply:
Of course. I thought most of that was already in there.

Reply:
Created #262 for this.